go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/weaveworks/kured
 go 1.15
 
 require (
+    github.com/containrrr/shoutrrr v0.4.1
```

Comment:
Is this indented correctly?

Reply:
for consistency we want this new entry to be indented by one tab (not four spaces)